modules/AWS/rds/locals.tf

```diff
@@ -1,6 +1,4 @@
 locals {
-  db_name = var.snapshot_identifier == null ? var.product : null
```

Comment:
Removing this condition to avoid confusion. We should recommend users to take snapshot from terraform environment

Reply:
Thats not bad idea, I removed this condition because if users take snapshot from terraform environment the snapshot will have product name as db_name and there is no way to workaround it. I think it is better let user configure it